src/lib/ng-v5/entry-point/write-package.transform.ts

```diff
@@ -60,6 +60,18 @@ export async function writePackageJson(entryPoint: NgEntryPoint, binaries: { [ke
     packageJson[fieldName] = binaries[fieldName];
   }
 
+  // read tslib version from our peerDependencies
+  if (!(packageJson.dependencies && packageJson.dependencies.tslib)) {
+    const tslibVersion = fs.readJsonSync(path.resolve('./package.json')).peerDependencies.tslib;
```

Comment:
This may break in a user install when trying to read package.json from current working directory.

Can we re-write to utilize node resolution like this?

```ts
const TSICKLE_VERSION = require('tsickle/package.json').version;
```

Reply:
Good catch, I'll fix it soon

Reply:
I actual had to change it a bit a left an in code comment